### DISCUSSION TITLE
Support gRPC web

### DISCUSSION THREAD

**Discussion by phamvinhdat** [refinedev/refine#3470] [Integration request]
### Is your feature request related to a problem? Please describe.

GRPC is everywhere, so It's good if we have an official data provider for grpc web.

### Describe the thing to improve

Can we have an official data provider for grpc web?

**Comment by omeraplak** [comment]
Hey @phamvinhdat ,
Thank you for contacting us! It would be great to see what community can do ⚡️

**Comment by Oelyhasina** [comment]
hello, is there already someone who has customized a dataprovider for api trpc (next js)? THANKS

**Comment by kenbrawn** [comment]
Hi! Can we have an example of how to customize a data provider with TRPC to refine (next js)?

**Comment by Hessuew** [comment]
Also looking for the trpc example and solutions
